exporter/importables.go

```diff
@@ -1990,31 +1989,36 @@ var resourcesMap map[string]importable = map[string]importable{
 			return nil
 		},
 		Import: func(ic *importContext, r *resource) error {
-			var share tf_sharing.ShareInfo
-			s := ic.Resources["databricks_share"].Schema
-			common.DataToStructPointer(r.Data, s, &share)
-			// TODO: how to link recipients to share?
+			// Handle share objects by reading them directly from the data
+			// since the resource has been migrated to pluginfw
 			ic.emitUCGrantsWithOwner("share/"+r.ID, r)
-			for _, obj := range share.Objects {
-				switch obj.DataObjectType {
+
+			// Get objects array from the resource data
+			objectsList := r.Data.Get("object").([]any)
```

Comment:
`r.Data` will still be using the SDKv2 implementation under the hood, so I'm not sure if there is a benefit of using `.Get` to get raw data instead of doing `common.DataToStructPointer`